docs/HOWTO_HEROKU_DEPLOY.md

```diff
@@ -16,23 +18,34 @@
   - Domain name = AUTH0_DOMAIN
   - Client ID = AUTH0_CLIENT_ID
   - Client Secret = AUTH0_CLIENT_SECRET
-- These variables should be placed in your heroku environment variables form
-- Scroll to `Allowed Callback URLs` section and update it with (your heroku_app_url):
+- These variables should be placed in your Heroku configuration variables form
+- Scroll to `Allowed Callback URLs` section and update it with (your HEROKU_APP_URL):
   - `https://<YOUR_HEROKU_APP_URL>/login-callback, http://<YOUR_HEROKU_APP_URL>/login-callback`
 
-- Scroll to `Allowed Logout URLs` section and update it with (your heroku_app_url):
-  - `https://<YOUR_HEROKU_APP_URL>/login-callback, http://<YOUR_HEROKU_APP_URL>/login-callback`
-
-  - `https://<YOUR_HEROKU_APP_URL>/login-callback` = AUTH0_LOGIN_CALLBACK
-  - `https://<YOUR_HEROKU_APP_URL>/logout-callback` = AUTH0_LOGOUT_CALLBACK
+- Scroll to `Allowed Logout URLs` section and update it with (your HEROKU_APP_URL):
+  - `https://<YOUR_HEROKU_APP_URL>/logout-callback, http://<YOUR_HEROKU_APP_URL>/logout-callback`
+- As a note:
```

Comment:
We don't have to do this for this PR, but it would be good to remove these environment variables altogether and just use `BASE_URL`.

I might remember saying the opposite at some point earlier in the project, but I'm convinced now that the fewer variables the better, now.